src/neuroconv/datainterfaces/ophys/miniscope/miniscopedatainterface.py

```diff
@@ -0,0 +1,63 @@
+import os
+from typing import Optional
+from glob import glob
+
+from natsort import natsorted
+from pynwb.file import NWBFile
+from pynwb.image import ImageSeries
+
+from ....basedatainterface import BaseDataInterface
+from ....tools.nwb_helpers import make_or_load_nwbfile
+
+
+class MiniscopeImagingInterface(BaseDataInterface):
+    def __init__(self, data_dir: str):
+        super().__init__(data_dir=data_dir)
```

Comment:
```suggestion
    def __init__(self, folder_path: FolderPathType):
        super().__init__(folder_path=folder_path)
```

using `from neuroconv.utils import FolderPathType`